### PR TITLE
BC-5522 - add var for the admin server port (part of the Deletion API implementation)

### PR DESCRIPTION
**Pull request by bn-pass**
# Description

## Links to Tickets or other pull requests

https://ticketsystem.dbildungscloud.de/browse/BC-5522

## Approval for review

- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

**Review by WojciechGrancow — APPROVED** [review]

File: ansible/group_vars/brb/instance_cfg.yml [hunk 1]
```diff
@@ -51,6 +51,7 @@ SERVER_LDAP_SYNC_FULL_CRONJOB: "0 */1 * * *"
 WITH_LDAP: true
 # PORTS
 PORT_SERVER: 3030
+PORT_ADMIN_SERVER: 4030
```
Comment (mamutmk5) [inline ansible/group_vars/brb/instance_cfg.yml:54]: Where is it used if no where then delet it

**Review by mamutmk5 — CHANGES_REQUESTED** [review]

File: ansible/group_vars/brb/instance_cfg.yml (hunk 1 above)
Comment (sszafGCA) [reply ansible/group_vars/brb/instance_cfg.yml:54]: I think it can be deleted

**Review by mamutmk5 — CHANGES_REQUESTED** [review]

File: ansible/host_vars/ref-audit/pod.yml [hunk 2]
```diff
@@ -22,3 +22,4 @@ STORAGE_REPLICAS: 1
 ERWINIDM_REPLICAS: 3
 TLDRAW_CLIENT_REPLICAS: 3
 TLDRAW_SERVER_REPLICAS: 3
+ADMIN_API_SERVER_REPLICAS: 3
```
[the remark this answers is not in the document]
Comment (sszafGCA) [reply ansible/host_vars/ref-audit/pod.yml:25]: done

File: ansible/host_vars/ref-nbc/pod.yml [hunk 3]
```diff
@@ -26,3 +26,4 @@ AMQP_FILE_PREVIEW_MIN_REPLICA_COUNT: 1
 AMQP_FILE_PREVIEW_MAX_REPLICA_COUNT: 5
 TLDRAW_CLIENT_REPLICAS: 3
 TLDRAW_SERVER_REPLICAS: 3
+ADMIN_API_SERVER_REPLICAS: 3
```
[the remark this answers is not in the document]
Comment (sszafGCA) [reply ansible/host_vars/ref-nbc/pod.yml]: done

**Review by mamutmk5 — APPROVED** [review]

**Review by mamutmk5 — APPROVED** [review]

**Pull request merged**